dll/win32/msv1_0/msv1_0.c

```diff
@@ -1785,24 +1783,24 @@ SpUserModeInitialize(
 
     *PackageVersion = SECPKG_INTERFACE_VERSION;
 
-    RtlZeroMemory(&Tables, sizeof(Tables));
-
-//    Tables[0].InstanceInit = SpInstanceInit;
-//    Tables[0].InitUserModeContext = NULL;
-//    Tables[0].MakeSignature = NULL;
-//    Tables[0].VerifySignature = NULL;
-//    Tables[0].SealMessage = NULL;
-//    Tables[0].UnsealMessage = NULL;
-//    Tables[0].GetContextToken = NULL;
-//    Tables[0].SpQueryContextAttributes = NULL;
-//    Tables[0].CompleteAuthToken = NULL;
-//    Tables[0].DeleteUserModeContext = NULL;
-//    Tables[0].FormatCredentials = NULL;
-//    Tables[0].MarshallSupplementalCreds = NULL;
-//    Tables[0].ExportContext = NULL;
-//    Tables[0].ImportContext = NULL;
-
-    *ppTables = Tables;
+    RtlZeroMemory(NtlmUsrFn, sizeof(NtlmUsrFn));
```

Comment:
You can probably use C99 structure assignment here. It will fill all necessary fields and leave everything else with zeroes

Reply:
done.